search-app/test/cmr/search/test/results_handlers/results_handler_util.clj

```diff
@@ -11,8 +11,8 @@
                                    :associated-concept-id "C1200000007-PROV1",
                                    :data {:convert-format {:XYZ "ZYX"}, :allow-regridding "true"}}]
                        :tools [{:tool-concept-id "TL1200000010-PROV1", :associated-concept-id "C1200000007-PROV1"}] ,
-                       :dataqualitysummaries ["DQS1200000012-PROV1"],
-                       :orderoptions ["OO1200000014-PROV1" "OO1200000015-PROV1"]}
+                       :dataqualitysummaries [{:concept-id "DQS1200000012-PROV1"}],
+                       :orderoptions [{:concept-id "OO1200000014-PROV1"} {:concept-id "OO1200000015-PROV1"}]}
          expected-detail {:variables [{:concept-id "V1200000019-PROV1"} {:concept-id "V1200000021-PROV1"}],
                           :services [{:data {:convert-format {:XYZ "ZYX"}, :allow-regridding "true"}, :concept-id "S1200000009-PROV1"}],
                           :tools [{:concept-id "TL1200000010-PROV1"}] ,
```

Comment:
Could you add some tests that show the revision-ids in the expected-detail?  It's actually not obvious to me where you get the revision-id info in association details.

Reply:
When you added payload to collections, services, services and generics, are they through both the current collection/service, collection/tool association interface, and the new generic association interface?

Reply:
Yes